Singapore/src/main/java/com/infoshareacademy/App.java

```diff
@@ -1,13 +1,23 @@
 package com.infoshareacademy;
 
+
+import com.infoshareacademy.model.User;
+import com.infoshareacademy.service.Opinion;
+
 /**
  * Hello world!
- *
  */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+public class App {
+    public static void main(String[] args) {
```

Comment:
Na  ten moment myślę, że jednego usera sobie "zahardkodujemy" to znaczy zrobimy na sztywno, a zaczniemy działać z resztą funkcjonalności